conan/api/model/remote.py

```diff
@@ -36,4 +36,9 @@ def __repr__(self):
         return str(self)
 
     def invalidate_cache(self):
+        """ If external operations might have modified the remote since it was instantiated,
```

Comment:
While I was moving things around in the docs (PR to follow), noticed that user might want to run this manually, so made this public